README.md

```diff
@@ -7,25 +7,78 @@
 
 ## Group Members
 
+* Justin Hart
+* Devin Gray
+* Derek Kramer
+* Tristan Gilfoyle
+* Devin Hanaway
+* Izabela louw
 
 ## Project Description
 
+MicroLens is consumer facing food tracker, that allows users to track the food they eat, based upon Micro/Macro Nutrients <> compared to the DRI (Daily Recommended Intake). We will allow users to input food, take a photo of a foods bar code, and as a stretch take a photo of a food and return the nutritional data for it.
+
+We will give the users a daily graph of their micro and macro nutrients compared to the DRI values.
+
 
 ## Who uses it?
 
+People who currently track their food intake, but would like to have a more in depth breakdown of their nutrients.
+
+People who have a chronic or acute condition, and would like a recommended nutritional plan that helps their disposition.
 
 ## What outputs do they need?
 
+ * Micro Nutrients from the food they eat.
+
+ * Macro Nutrients from the food they eat.
+
+ * Comparison between a users micro nutrient intake with Daily Recommended Intake.
 
 ## What inputs are needed to generate those outputs?
 
+ * USDA OPEN FOOD API
```

Comment:
the API isn't really an input- you're receiving info from that as a developer, but we usually think of input/outputs as the relationship between you and the customer